joomla/joomla-cms
# [4.0] Update UCM table classes

Followup to https://github.com/joomla/joomla-cms/pull/27992.

### Summary of Changes

Updates remaining `#__content_types` entries with fully qualified class names.

Note, this also changes article table from `Joomla\CMS\Table\Content` to `Joomla\Component\Content\Administrator\Table\ArticleTable`. I'm actually not sure if this is needed or not, or why this class was added to 4.0 in the first place. Need feedback on this.

### Testing Instructions

This will need to be tested both for new installations and for upgrades from 3.10.
Create and edit some categories for different content types.
Create and edit some articles.
In edit form click Versions button. Check that current item version is marked with a star.

### Documentation Changes Required

No.
